site-src/guides/index.md

```diff
@@ -128,12 +215,13 @@ This quickstart guide is intended for engineers familiar with k8s and model serv
    The following cleanup assumes you would like to clean ALL resources that were created in this quickstart guide.  
    please be careful not to delete resources you'd like to keep.
    ```bash
-   kubectl delete -f https://github.com/kubernetes-sigs/gateway-api-inference-extension/raw/main/config/manifests/gateway/traffic_policy.yaml --ignore-not-found
-   kubectl delete -f https://github.com/kubernetes-sigs/gateway-api-inference-extension/raw/main/config/manifests/gateway/extension_policy.yaml --ignore-not-found
-   kubectl delete -f https://github.com/kubernetes-sigs/gateway-api-inference-extension/raw/main/config/manifests/gateway/patch_policy.yaml --ignore-not-found
+   kubectl delete -f https://github.com/kubernetes-sigs/gateway-api-inference-extension/raw/main/config/manifests/gateway/gke/gateway.yaml --ignore-not-found
```

Comment:
Can we please split it into three please: 
- `Uninstall the InferencePool` (uninstalls the InferencePool and InferenceModels)
- `Uninstall the gateway` (uninstalls the httproute and gateway)
- `Uninstall the APIs` (uninstalls the CRDs)